[No QA] Docs updates for E/A#94131 surface RuleBot in Agent rules section (#94636)
Expensify/App: docs/articles/new-expensify/ai-agents/Create-Agent-Rules.md

When you create your first Agent Rule, Expensify automatically creates RuleBot and adds it to the workspace as a Workspace Admin.

> Once RuleBot is created, the **Agent rules** section displays the line **Agent rules are enforced by** followed by RuleBot, so you can see which agent enforces your rules. This line only appears after RuleBot has been created, so it isn't shown before you add your first Agent Rule.

Reviewer: ```suggestion
After RuleBot is created, the **Agent rules** section displays **Agent rules are enforced by** followed by **RuleBot**. This indicates which agent enforces all Agent rules in the workspace.
```
